### PR TITLE
Change TempDir to const so functions are gcsafe

### PR DESCRIPTION
**Pull request by inverimus**

### CODE REVIEW

File: src/xlsx/utils.nim [hunk 1]
```diff
@@ -10,8 +10,7 @@ const
   UpperLetters = {'A' .. 'Z'}
   CharDataOption = {xmlCharData, xmlWhitespace}
   DEFAULT_WORKBOOK_PATH = "xl/workbook.xml"
-let TempDir* = getTempDir() / "xlsx_windx_tmp" ## temp dir for all extracted xml files from Excel
-
+  XLSX_TMP = "xlsx_windx_tmp"
```
Comment (ringabout) [inline src/xlsx/utils.nim:13]: You can use `const` instead, since `getTempDir` works at compile time.

Comment (inverimus) [reply src/xlsx/utils.nim]: Yes, that's better. I had just assumed it couldn't be since it wasn't already.

File: xlsx.nimble [hunk 2]
```diff
@@ -27,6 +27,6 @@ task docs, "Generate docs":
     "--git.commit:master " &
     "--index:on " &
     "--git.devel:master " &
-    "--git.url:https://github.com/xflywind/xlsx " &
+    "--git.url:https://github.com/gcaliber/xlsx " &
```
Comment (ringabout) [inline xlsx.nimble:30]:
```suggestion
    "--git.url:https://github.com/ringabout/xlsx " &
```

**Review by ringabout — APPROVED** [review]
Nice, thank you!

**Pull request merged**